### ISSUE TITLE
More clarification on how to get started with Objective-C?

### ISSUE THREAD

**Issue by BeggarsMattM** [exercism/exercism.io#987]
With the proviso that I understand that Exercism may be primarily for "people who already have some facility in a language" to improve their style in that language, and not a tool to learn that language from scratch...

I am completely at a loss about how to "set up Objective-C" based on the information given in the section of that name.

I can fire up Xcode but am immediately presented with a baffling array of menus and options.  So should I definitely already know how to generate and work on the right kind of Xcode project here?

I installed objc and xctool and tried that out.  I got no information other than "*\* TEST FAILED: 0 of 0 tests passed **" - I saw the warning that these messages would be less useful than the ones in Xcode, but... no useful information at all?  I tried copying what is presumably a working solution from this repository and got exactly the same "0 of 0 tests passed" error message, plus a mystifying "target specifies product type 'com.apple.product-type.bundle.unit-test', but there's no such product type for the 'macosx' platform" message which checking dependencies in the test-suite now.

tl;dr: Can someone recommend a tutorial or other source of information that will help someone who isn't already a coder in Objective-C to reach the level of knowledge required to begin this track?
